kythe/go/platform/tools/extraction/extractrepo/extractrepo.go

```diff
@@ -89,11 +92,44 @@ func verifyFlags() {
 		log.Println("You must provide a non-empty -output")
 	}
 
+	if *tempRepoDir != "" {
+		if !empty(*tempRepoDir) {
+			hasError = true
+			log.Println("-tmp_repo_dir must be an empty directory.")
+		}
+	}
+
 	if hasError {
 		os.Exit(1)
 	}
 }
 
+func empty(dir string) bool {
```

Comment:
`isEmptyDir` maybe.

Please document this, too.

```
// isEmptyDir reports whether dir is an empty directory. If dir does not exist,
// an empty directory is created at that path.
```

or words to that effect

Reply:
done